db/utils.py

```diff
@@ -0,0 +1,30 @@
+from hashlib import md5
+import os.path
+import json
+
+def hashstring(text):
+    return md5(text.encode()).hexdigest()
+
+class Config(object):
+
+        def __init__(self, path='config.json'):
+            self.defaultconfig = {'db_url':'sqlite:///:memory:', 'debug':True}
+
+            if not os.path.isfile(path):
+                print("No config.json found, creating default config.")
+                with open(path, 'w') as conf:
+                    conf.write(json.dumps(self.defaultconfig))
+            with open(path, 'r') as conf:
+                self.config = json.loads(conf.read())
+        
+        def getconfig(self, handle):
+                return self.config.get(handle)
+
```

Comment:
Nifty. Only, now I feel like I should just be subclassing dict to make this config class. Nothing wrong with that I guess.